diffengine/__init__.py

```diff
@@ -99,7 +104,7 @@ class Entry(BaseModel):
     url = CharField()
     created = DateTimeField(default=datetime.utcnow)
     checked = DateTimeField(default=datetime.utcnow)
-    tweet_status_id_str = CharField(null=True)
+    tweet_status_id_str = CharField(null=False, default="")
```

Comment:
Hey @andresfib this breaks the tweeting functionality. Can this be `CharField(null=True)`. 

The thread creation is based in this to be `None` the first time and now it is `''` so the thread is not created.

You don't use this in the sendgrid functionality so I wonder why this was donde this way.

The test doesn't break because it mocks this as `None` so it passes anyway...